packages/flutter/lib/src/foundation/tree_diagnostics_mixin.dart

```diff
@@ -6,6 +6,16 @@ import 'package:meta/meta.dart';
 
 import 'print.dart';
 
+/// Returns a 5 character long hexadecimal string generated from
+/// Object.hashCode's 20 least-significant bits.
```

Comment:
While you're here, it might be worth pointing to `[descrimeIdentity]` as well, either in a "See also" block, or just a separate paragraph that talks about their relationship.